pandaserd/module.py

```diff
@@ -27,7 +34,9 @@ def __init__(self, table, table_name, **kwargs):
         self.align = 'left'
         self.font_color = 'grey60'
         self.bg_color = kwargs.get('bg_color', 'grey')
-        bg_colors = ['lightblue', 'skyblue', 'pink', 'lightyellow', 'grey', 'gold']
+        bg_colors = self.html_colors #['lightblue', 'skyblue', 'pink', 'lightyellow', 'grey', 'gold']
+        # TODO: Set font_color complementary to bg_color
```

Comment:
Would be nice indeed! Two methods I can think if (perhaps you already have an idea how to do this): 
1. you can find the RGB of a complementary color from using [this method](https://stackoverflow.com/a/37675777) and then find the closest available color in the `colors.json` file.
2. you can create a dictionary once that lists the complementary color for each key in `colors.json` and do a lookup there.